docs/api-reference.rst

```diff
@@ -2,6 +2,21 @@
 API Reference
 *************
 
+The API reference gives an overview of Geomstats implementation.
+
+The module `geometry` implements concepts in differential geometry,
+such as manifolds and Riemannian metrics, with associated exponential
+and logarithmic maps, geodesics, and parallel transport.
+
+The module `learning` implements statistics and learning algorithms for data
+on manifolds, such as estimation, clustering and dimension reduction.
+The code is object-oriented and classes inherit from
+scikit-learn's base classes and mixins.
+
+In both modules, the operations are vectorized for batch computation and provide
+support for different execution backends---namely NumPy, PyTorch, and TensorFlow---
+enabling GPU acceleration. The module `backend` implements the operations needed to use Geomstats seemlessly with any backend.
```

Comment:
Is GPU acceleration tested anywhere?

Reply:
Not really, although if pytorch and tf backends work, then we should be able to use GPU and therefore use their GPU acceleration. I'll remove it here and in the JMLR paper although, to avoid being criticized on this.